packages/amplify-e2e-core/src/categories/lambda-layer.ts

```diff
@@ -1,11 +1,12 @@
+import { JSONUtilities } from 'amplify-cli-core';
 import fs from 'fs-extra';
 import _ from 'lodash';
 import path from 'path';
 import { nspawn as spawn, ExecutionContext, getCLIPath, KEY_DOWN_ARROW } from '../../src';
 import { getLayerVersion, listVersions } from '../utils/sdk-calls';
 import { multiSelect } from '../utils/selectors';
 
-type LayerRuntimes = 'dotnetcore3.1' | 'go1.x' | 'java' | 'nodejs' | 'python';
+export type LayerRuntimes = 'nodejs' | 'python';
```

Comment:
Are we removing official support for using LL with those other runtimes? If so, do we have a docs PR for that limitation?

Reply:
We've only ever supported nodejs and python.